### PR TITLE
Terms::setMissingBucket()

### PR DESCRIPTION
**Pull request by andriinavrotskii**

### CODE REVIEW

File: tests/Aggregation/TermsTest.php [hunk 1]
```diff
@@ -162,4 +162,15 @@ private function getIndex(): Index
 
         return $index;
     }
+
+    /**
+     * @group functional
```
Comment (ruflin) [inline tests/Aggregation/TermsTest.php:167]: This does not look like a functional test. As far as I can see, no call is made to Elasticsearch. Any chance you could expand the test in a way that a call to Elasticsearch is made to confirm the functionality.

**Review by ruflin — COMMENTED** [review]
Thanks for the contribution. Left one comment around the tests.

File: tests/Aggregation/TermsTest.php (hunk 1 above)
Comment (andriinavrotskii) [reply tests/Aggregation/TermsTest.php]: You are right. My mistake. Updated.

**Review by ruflin — APPROVED** [review]

**Comment by ruflin** [comment]
Thanks for the change!

**Pull request merged**